types/node/index.d.ts

```diff
@@ -1514,7 +1543,6 @@ declare module "https" {
         ciphers?: string;
         honorCipherOrder?: boolean;
         requestCert?: boolean;
-        rejectUnauthorized?: boolean;
```

Comment:
This is already in place but in https it's still named `RequestOptions`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/index.d.ts#L1522